sumo_web3d/server/server.py

```diff
@@ -350,36 +356,49 @@ def start_sumo_executable(gui, sumo_args, sumocfg_file):
 def simulate_next_step():
     global last_lights, last_vehicles
     start_secs = time.time()
+    # Vehicles
```

Comment:
Why move this? Admittedly this is a cryptic comment, could be something like "Update Vehicles" and "Update Lights."